### PR TITLE
Support new axios https client in monero-ts

### PR DESCRIPTION
**Pull request by mainnet-pat**

### CODE REVIEW

File: src/offline_wallet_generator.ts [hunk 1]
```diff
@@ -12,7 +12,12 @@ async function main() {
   let myNum: number = 0;
 
   let manager = new moneroTs.MoneroConnectionManager();
-  await manager.setConnection("http://localhost:38081");
+  await manager.setConnection({
```
Comment (woodser) [inline src/offline_wallet_generator.ts:15]: Please undo this change with authentication.

File: src/sample_code.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ async function runMain() {
   
   // connect to a daemon
   console.log("Connecting to daemon");
-  let daemon = await moneroTs.connectToDaemonRpc("http://localhost:28081");
+  let daemon = await moneroTs.connectToDaemonRpc("http://localhost:28081", "superuser", "abctesting123");
```
Comment (woodser) [inline src/sample_code.ts:12]: Please undo change with authentication.

File: src/offline_wallet_generator.ts (hunk 1 above)
Comment (woodser) [reply src/offline_wallet_generator.ts:15]: This one still needs undone.

**Comment by mainnet-pat** [comment]
done
